src/v/cloud_storage/partition_manifest.cc

```diff
@@ -263,12 +263,12 @@ const model::offset partition_manifest::get_last_offset() const {
 }
 
 const std::optional<kafka::offset>
-partition_manifest::get_last_kafka_offset() const {
+partition_manifest::get_next_kafka_offset() const {
     auto last_seg = last_segment();
```

Comment:
`partition_manifest::last_segment` doesn't exist on this version. Did you mean to add it?

Reply:
Nice spot, this was unintentional. I just removed it.